cluster/olm/ceph/generate-rook-csv-templates.sh

```diff
@@ -2,9 +2,6 @@
 set -e
 
 export OLM_SKIP_PKG_FILE_GEN="true"
-export OLM_INCLUDE_CEPHFS_CSI="true"
```

Comment:
I can see value in having this but at the same time, OLM is so rigid that we want to enable CEPHFS CSI post-deployment I'm not sure it would even be possible so I'm fine with removing this.